make-file/installer.mk

```
@@ -0,0 +1,38 @@
+.PHONY: installer-lint-back
+installer-lint-back: #Doc: launch PHPStan for installer
+	$(PHP_RUN) vendor/bin/phpstan analyse --configuration src/Akeneo/Platform/Installer/back/tests/phpstan.neon
+	${PHP_RUN} vendor/bin/php-cs-fixer fix --diff --dry-run --config=src/Akeneo/Platform/Installer/back/tests/.php_cs.php src/Akeneo/Platform/Installer/back/src
+	${PHP_RUN} vendor/bin/rector process --dry-run --config=src/Akeneo/Platform/Installer/back/tests/rector.php
+
+.PHONY: installer-lint-fix-back
+installer-lint-fix-back: #Doc: launch PHP CS fixer for installer
+	${PHP_RUN} vendor/bin/php-cs-fixer fix --diff --config=src/Akeneo/Platform/Installer/back/tests/.php_cs.php src/Akeneo/Platform/Installer/back/src
```

We should not override the path defined in the config file
```suggestion
	${PHP_RUN} vendor/bin/php-cs-fixer fix --diff --config=src/Akeneo/Platform/Installer/back/tests/.php_cs.php 
```